js/audio/interactive/InteractiveAudio.svelte

```diff
@@ -201,7 +201,7 @@
 		dispatch("start_recording");
 		if (!inited) await prepare_audio();
 		header = undefined;
-		if (streaming) {
+		if (streaming && recorder.state != "recording") {
```

Comment:
Sometimes this would get triggered twice and there would be an exception in the console since the recorder was already recording. But it wouldn't break anything.